Track/Track.py

```diff
@@ -907,22 +908,31 @@ def updateParameterNodeFromGUI(self, caller=None, event=None):
         # Segmentation file should end with .mha
         segmentationNode = slicer.util.loadVolume(self.selector3DSegmentation.currentPath,
                                                   {"singleFile": True, "show": False})
-        self.logic.clearSliceForegrounds()
-        segmentationNode.SetName("3D Segmentation")
-        # Set a param to hold the 3D segmentation node ID
-        nodeID = shNode.GetItemByDataNode(segmentationNode)
-        self.customParamNode.node3DSegmentation = nodeID
-
-        # Create a label map of the 3D segmentation that will be used to define the mask overlayed
-        # on the 2D images during playback
-        volumesModuleLogic = slicer.modules.volumes.logic()
-        segmentationLabelMap = slicer.mrmlScene.AddNewNodeByClass('vtkMRMLLabelMapVolumeNode',
-                                                                  "3D Segmentation Label Map")
-        volumesModuleLogic.CreateLabelVolumeFromVolume(slicer.mrmlScene, segmentationLabelMap,
-                                                       segmentationNode)
-        # Set a param to hold the 3D segmentation label map ID
-        labelMapID = shNode.GetItemByDataNode(segmentationLabelMap)
-        self.customParamNode.node3DSegmentationLabelMap = labelMapID
+        if np.unique(slicer.util.arrayFromVolume(segmentationNode)).size == 2:
```

Comment:
can you add a comment here? what is this check about?

Reply:
Added! The check is to ensure segmentation files are a binary mask